autoware_launch/launch/autoware.launch.xml

```diff
@@ -100,8 +100,9 @@
 
   <!-- Perception -->
   <group if="$(var launch_perception)">
-    <include file="$(find-pkg-share perception_launch)/launch/perception.launch.xml">
-      <arg name="mode" value="$(var perception_mode)" />
+    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_perception_component.launch.xml">
+      <arg name="mode" value="lidar"/>
```

Comment:
Is it intentional to set the mode to lidar instead of $(var perception_mode)?

Reply:
Thank you, that is a mistake. Fixed here: https://github.com/tier4/autoware_launch/pull/681/commits/bf9a1ae23b6d4a743d7dfbd755ca2e28591cd9d0